eng/common/scripts/get-diff-changes-from-PR.ps1

```diff
@@ -0,0 +1,32 @@
+
+function Get-PullRequest-Diff-Changes {
```

Comment:
```suggestion
function Get-ChangedFilesForPullRequest {
```

Reply:
I was debating whether or not we should tie this to PR or not but other than the default values there is nothing that ties this functionality to only PRs.

Reply:
Done